src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin.Test/Commands.AzureStackAdmin.Test.csproj

```diff
@@ -0,0 +1,198 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\packages\xunit.runner.visualstudio.2.1.0-beta4-build1109\build\net20\xunit.runner.visualstudio.props" Condition="Exists('..\..\..\packages\xunit.runner.visualstudio.2.1.0-beta4-build1109\build\net20\xunit.runner.visualstudio.props')" />
```

Comment:
xunit should be xunit.runner.visualstudio.2.1.0 version. Please look to other test projects in the ReousreManager folder.